src/include/sof/audio/component.h

```diff
@@ -634,6 +634,64 @@ struct comp_dev {
 #endif
 };
 
+/**
+ * Get a pointer to a first comp_buffer object providing data to the component
```

Comment:
not a native English speaker, but isn't it "the first?"

Reply:
@lyakh you are correct